src/lando/settings.py

```diff
@@ -214,13 +214,19 @@
 CELERY_ACCEPT_CONTENT = ["json"]
 CELERY_TASK_SERIALIZER = "json"
 
-DEFAULT_FROM_EMAIL = "Lando <lando@lando.test>"
+LANDO_USER_NAME = os.getenv("LANDO_USER_NAME", "Lando")
+LANDO_USER_EMAIL = os.getenv("LANDO_USER_EMAIL", "lando@lando.test")
+
+DEFAULT_FROM_EMAIL = f"{LANDO_USER_NAME} <{LANDO_USER_EMAIL}>"
```

Comment:
Bikeshed: since these new settings are all Git-specific, maybe we should reflect that in the names, like `LANDO_GIT_USER_NAME`, `LANDO_GIT_COMMITTER_NAME` etc?

Reply:
The `DEFAULT_FROM_EMAIL` constant pre-existed the git implementation, so I split it so I could reuse the components. [Looking through the code now, though, I don't see any use of it, except in a test].

I ain't painting it green! (unless you insist)